### ISSUE TITLE
Figure out which commit from upstream web-core introduces the compile error

### ISSUE THREAD

**Issue by nicolasbrugneaux** [celo-org/web-core#6]

**Comment by nicolasbrugneaux** [comment]
Example log: https://vercel.com/c-labs/web-core/AqSr1sLtoDumak21hf4zgzKhHNrP


Meaningful log:
```
$ next build
--
18:19:21.238 | warn  - You have defined experimental feature (modularizeImports) in next.config.mjs that does not exist in this version of Next.js.
18:19:21.238 | warn  - Please remove it from your configuration.
18:19:21.238 |  
18:19:21.238 | warn  - You have enabled experimental feature (images) in next.config.mjs.
18:19:21.238 | warn  - Experimental features are not covered by semver, and may cause unexpected or broken application behavior. Use at your own risk.
18:19:21.238 |  
18:19:21.335 | info  - Linting and checking validity of types...
18:19:48.002 | Failed to compile.
18:19:48.003 |  
18:19:48.003 | Type error: Type instantiation is excessively deep and possibly infinite.
18:19:48.003 |  
18:19:48.044 |  
18:19:48.044 | > Build error occurred
18:19:48.046 | Error: Call retries were exceeded
18:19:48.046 | at ChildProcessWorker.initialize (/vercel/path0/node_modules/next/dist/compiled/jest-worker/index.js:1:11661)
18:19:48.046 | at ChildProcessWorker._onExit (/vercel/path0/node_modules/next/dist/compiled/jest-worker/index.js:1:12599)
18:19:48.046 | at ChildProcess.emit (node:events:513:28)
18:19:48.046 | at Process.ChildProcess._handle.onexit (node:internal/child_process:293:12) {
18:19:48.046 | type: 'WorkerError'
18:19:48.046 | }
18:19:48.062 | error Command failed with exit code 1.
18:19:48.063 | info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
18:19:48.078 | Error: Command "yarn run build" exited with 1

**Comment by nicolasbrugneaux** [comment]
The issue doesn't exist on `celo-org/feat/celo` but appears as soon as merging `safe-global/main` into it, so we need to bisect all commits and figure out the culprit...

**Comment by nicolasbrugneaux** [comment]
We didn't figure it out. However, we found a more sustainable solution. We swapped from the vercel pipelines to using the same github actions as the upstream repo. It should prove more resilient. Closed by 1ba698e6 and db619e61
